komodo-core-tests/pom.xml

```diff
@@ -58,7 +58,7 @@
 
 		<dependency>
 			<groupId>org.mockito</groupId>
-			<artifactId>mockito-core</artifactId>
+			<artifactId>mockito-all</artifactId>
```

Comment:
In the commit message, can you specify the reason you need all of mockito rather than just the core. Running most of the teiid unit tests did not seem to require anything more than mockito-core. Either way, if you need all of it, no problem - just specify reason in commit.

Reply:
I'm not sure the commit message is the right place to cover every change in the commit set.  This is to align with jboss-parent, which uses all rather than core.

Reply:
The commit message is absolutely the place to record such changes. Its the only log that remains with the source code. It should never be necessary to refer to an alternative source to understand decisions for a specific code change. Certainly, common coding standards should require that some description be added to a commit message.